Documentation/contributing.md

```diff
@@ -77,6 +77,7 @@ DOs and DON'Ts
   of time.
 * **DON'T** commit code that you didn't write. If you find code that you think is a good fit to add to .NET Core, file an issue and start a discussion before proceeding.
 * **DON'T** submit PRs that alter licensing related files or headers. If you believe there's a problem with them, file an issue and we'll be happy to discuss it.
+* **DO NOT** fix merge conflicts using a merge commit. Prefer `git rebase`.
```

Comment:
I am not sure about this @rowandh we attempted using rebase and always got problems, to approve this I would need you to do a small video meeting and demonstrate what you mean exactly.

Reply:
Happy to do that

Reply:
I found rebase worked well when on a project where we used branches on the main repo (ie all had commit rights) - When doing it via forks here on this project, I got myself into a mess a couple of times.  However, if you can show it is better and how to manage it, would be great to see a demo.

Reply:
Let's schedule a call when everyone's back and I'll run through how I've been doing this already.

As an example, on [this PR](https://github.com/stratisproject/StratisBitcoinFullNode/pull/1742) I had to rebase to master 3 times because it was in review for so long. But if you look at the history, you see no "Merge branch 'master' into ..." commits, and the diff maintains the original changeset.